io.github.ec_.Quake3e.OpenArena.appdata.xml

```diff
@@ -32,7 +32,14 @@
       This Flatpak was made to be a replacement of the original one released in 2012,
       as it contains bugs and annoyances not present in this engine.
       The gameplay is 100% compatible and if not, please make
-      an issue in our repo, we'll investigate it.
+      an issue in our repo, we'll investigate it
+    </p>
+    <p>
+      A configuration file is also available, which lets you modify CVARS however you want.
+      The OAX patch has been applied, which adds support for more screen resolutions and makes the game playable
+      in many modern systems, it also includes bugfixes and improvements. (to disable this patch modify the config file).
+      Beware that trying to connect to an OAX-patched server may not with unpatched clients. Connections to unpatched servers
```

Comment:
>may not with unpatched clients

Not sure, but it sounds to me that there is a word missing here.

Reply:
Good call, new PR underway